hydromt_wflow/workflows/soilgrids.py

```diff
@@ -755,19 +755,35 @@ def soilgrids_brooks_corey(
     return ds_out
 
 
-def soilgrids_sediment(ds, ds_like, usleK_method, logger=logger):
+def soilgrids_sediment(
+    ds: xr.Dataset,
+    ds_like: xr.Dataset,
```

Comment:
I find these names really uninformative, but given that they were already like that, they can stay for now, but in future I would have prefered something like `ds_soil` and `ds_model` or something like that to make the code more readable

Reply:
These are legacy names, they should be renamed at some point but that is out of scope for this PR